sources/packages/api/src/utilities/notification-utils.ts

```diff
@@ -0,0 +1,2 @@
+export const STUDENT_FILE_UPLOAD_TEMPLATE_ID =
```

Comment:
Please create a constant file in the same scope of the service to keep this. Alternatively, if we consider it as a "configuration" it can also be moved to system-`configurations-constants.ts`.